### DISCUSSION TITLE
please add language. 'Korean'

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#5858] [General]
Name[ko]=MMEX Money Manager

GenericName[ko]=개인 재무 및 자금 관리 소프트웨어

Comment[ko]=재무 파악 및 현금 흐름 추적

Categories[ko]=재무;오피스;

**Comment by vomikan** [comment]
https://crowdin.com/project/moneymanagerex/ko

**Reply by DevAnyKR** [reply]
I have finished translating the application. The rest should be calibrated while actually using it.
Do I have to complete the help translation to apply to the next update? 😱
